modeltranslation/translator.py

```diff
@@ -77,7 +77,11 @@ class FieldsAggregationMetaClass(type):
     fields: Sequence[str]
 
     def __new__(cls, name: str, bases: tuple[type, ...], attrs: dict[str, Any]) -> type:
-        attrs["fields"] = set(attrs.get("fields", ()))
+        fields = attrs.get("fields", ())
+        if isinstance(fields, str):
+            attrs['fields'] = {fields}
```

Comment:
This is bad idea, it's clearly an error, so maybe throw `ImproperlyConfigured` here?